### DISCUSSION TITLE
Change Modal title? And launch QuickAdd from desktop shortcut?

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [chhoumann/quickadd#551] [Q&A]
I'm using QuickAdd to create a new file of a new dictionary entry and then place that in a folder. The modal that comes up is still called `fileName`. How can I change this?

And also: How can I launch this macro from a desktop shortcut?
I've tried this `obsidian://advanced-uri?vault=Dictionary%commandid=` but no matter which ID from QuickAdd's `data.json` I put there nothing happens besides opening the vault.

Any advice?

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that. Answering both parts:

**Modal title:** the prompt's title comes from the variable name in your choice's File Name Format. With the default (a bare `{{VALUE}}`) you get the generic file-name prompt. Enable **File Name Format** on the choice and name the variable whatever you want the modal to say:

```
{{VALUE:Dictionary entry}}
```

Now the prompt is titled "Dictionary entry". You can also add helper text under the title with `{{VALUE:Dictionary entry|label:The word you're defining}}`.

**Desktop shortcut:** you don't need Advanced URI - QuickAdd has its own URI handler:

```
obsidian://quickadd?vault=Dictionary&choice=<your choice name>
```

The `choice` parameter is the choice's *name* exactly as it appears in QuickAdd's settings (URL-encoded, so spaces become `%20`), not an id from `data.json` - that's why the command-id approach did nothing. You can also pre-fill prompts with `&value-<variable name>=...`. On Windows, a desktop shortcut can point directly at that `obsidian://` URL.

Docs: https://quickadd.obsidian.guide/docs/Advanced/ObsidianUri
